### ISSUE TITLE
DL4J/SameDiff: Add Layerwise Adaptive Rate Scaling (LARS)

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#7762]
See paper: https://openreview.net/pdf?id=rJ4uaX2aW 

This is mainly useful for large batch distributed training.

I don't like their exact formulation (introducing redundant/unnecessary 'trust' hyperparameter that is multiplied by base learning rate hyperparameter), but the overall idea is good and clearly effective for large batch training.

**Comment by agibsonccc** [comment]
Closing — SameDiff has been completely rewritten with a new DSP (Dynamic Shape Plan) execution engine, native plan compilation, CUDA graph replay, and Triton compilation. The architecture referenced in this issue no longer exists. If you still experience this issue with the upcoming release, please open a new issue.
